api/csgo.py

```diff
@@ -0,0 +1,161 @@
+import sqlite3
+from json.decoder import JSONDecodeError
+from typing import TYPE_CHECKING, Dict, List
+
+import requests
+
+from configuration import config
+
+if TYPE_CHECKING:
+    import telegram
+    import telegram.ext
+
+conn = sqlite3.connect('/db/steam_id.db', check_same_thread=False)
+cursor = conn.cursor()
+
+formula: str = f"CREATE TABLE IF NOT EXISTS `telegram_steam_id` ( " \
+               "`telegram_id` VARCHAR(255) NOT NULL UNIQUE, " \
+               "`steam_id` VARCHAR(255) NOT NULL, " \
+               "PRIMARY KEY (`telegram_id`))"
+cursor.execute(formula)
+
+STEAM_API_KEY = config["STEAM_API_KEY"]
+
+
+def make_csgo_response(username: str, steam_id: str) -> str:
+    url: str = "https://api.steampowered.com/ISteamUserStats/GetUserStatsForGame/v0002/"
+    params: Dict[str, str] = {"appid": "730", "key": STEAM_API_KEY, "steamid": steam_id}
+
+    try:
+        response: Dict[str, Dict] = requests.get(url, params).json()
+        username: str = username.replace('_', '\_')
```

Comment:
this is a bad idea, try Markdown_v2 parsemode

Reply:
Let's make this a separate PR.